sources/launcher/Stride.Launcher/Stride.Launcher.nuspec

```diff
@@ -5,7 +5,7 @@
     <version>5.0.6</version>
     <authors>Stride</authors>
     <owners>Stride</owners>
-    <licenseUrl>https://stride3d.net/license.txt</licenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Need to update this and the package nuspec below as this isn't proper for a nuspec.